compiler/luci-interpreter/pal/cmsisnn/PALQuantize.h

```diff
@@ -31,7 +31,7 @@ static inline void Quantize(tflite::QuantizationParams &params,
 
 template <typename Input, typename Output>
 static inline void Requantize(const Input *input_data, int32_t size,
-                              int32_t effective_scale_multiplier, int32_t effective_scale_shift,
+                              int32_t effective_scale_multiplier, int effective_scale_shift,
```

Comment:
Q) can you please share why we need this change? int will be different in 32 vs 64 bit architecture.